src/components/calculator.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './calculator.css';
+
+function Calculatore() {
```

Comment:
- [ ] According to the requirements, your ```Calculator``` function should have at least one child component. At the moment, your ```Calculator``` component only contains a tree of DOM elements (in this case, div and button elements) and does not render any other React components inside of it.